semantics/evaluate.lem

```diff
@@ -148,6 +148,9 @@ evaluate_match st env v ((p,e)::pes) err_v  =
     end
   else (st, Rerr (Rabort Rtype_error))
 
+declare {isabelle} rename function evaluate = fun_evaluate
+declare {isabelle} rename function evaluate_match = fun_evaluate_match
```

Comment:
There's a bug in Lem that makes it produce invalid Isabelle output when constants are automatically renamed. This works around that bug. (I'll open an issue on the Lem repository soon, when I have minimized it accordingly.)